tests/test_optimizer.py

```diff
@@ -32,57 +27,34 @@ def setUp(self):
             },
         }
 
+    def check_file(self, file, func=None, pretty=False, **kwargs):
+        module = getattr(sqlglot.optimizer, file)
+        func = getattr(module, func or file)
```

Comment:
So much magic haha
Can we just pass the function in?
It'd help us on dorky IDEs that have formal "refactor" features... not the hardcore folks who do find-and-replace